020-Connect/010-Postgres/001-direct-access.mdx

```diff
@@ -227,6 +227,23 @@ Tables created through the wire protocol will be missing the columns necessary t
 
 Please reference to the [special columns](/docs/concepts/data-model#special-columns) section to learn more about these columns.
 
+## Column metadata
+
+Xata uses SQL comments on columns to track some replication-related metadata about certain types of columns. These comments are only ever present on columns created using the Xata UI, and only on certain types of columns.
```

Comment:
Think it's more than the UI right? It's basically the Xata API. If I use the CLI for example or load from a prebuilt schema?

Reply:
The Xata CLI doesn't currently support adding columns to Postgres-enabled tables (via `xata schema edit`).

Restoring a database via `pg_restore` that contains these metadata comments implies that the original was created via the Xata UI, so I think the statement holds.